musicbrainzngs/mbxml.py

```diff
@@ -523,6 +525,9 @@ def parse_cdstub(cdstub):
 
     return result
 
+def parse_offset_list(ol):
```

Comment:
cool - it'd be nice to see xml tests for each method and some combination of their includes